projects/feray-polars/pyproject.toml

```diff
@@ -0,0 +1,39 @@
+[project]
+name = "feray-polars"
```

Comment:
if we agree to this structure- we have to request these package names and set them up in both pypi and test.pypi for passwordless publishing

Reply:
Cool yeah seems good to me, I'll defer to you and @danielgafni on naming and whatnot